### PR TITLE
TELCODOCS-430-rn

### PR DESCRIPTION
**Pull request by kdrake20**
Fixes: [TELCODOCS-430](https://issues.redhat.com/browse/TELCODOCS-430)

For: Version 4.11 

Signed-off-by: Katie Tothill ktothill@redhat.com

### CODE REVIEW

File: release_notes/ocp-4-11-release-notes.adoc [hunk 1]
```diff
@@ -199,6 +199,10 @@ The Raft algorithm allows for scaling of etcd members using a new `learner` stat
 [id=ocp-4-11-enabling-assisted-installer-service-on-bare-metal_{context}]
 {product-title} 4.11 introduces the `imageStorage` option as part of the `AgentServiceConfig` custom resource. This option improves application performance by allowing the user to specify persistent storage claim details for use with the image service.
 
+The `releaseImage` parameter of the `ClusterImageSet` custom resource now supports OS Image version identification. The discovery ISO is based on an OS image version as the releaseImage, or latest if the specfied version is unavailable.
```
Comment (tmalove) [inline release_notes/ocp-4-11-release-notes.adoc:202]:
```suggestion
The `releaseImage` parameter of the `ClusterImageSet` custom resource now supports operating system image version identification. The discovery ISO is based on an operating system image version as the releaseImage, or the latest version if the specified version is unavailable.
```

File: release_notes/ocp-4-11-release-notes.adoc [hunk 2]
```diff
@@ -199,6 +199,10 @@ The Raft algorithm allows for scaling of etcd members using a new `learner` stat
...
+
+The openshiftVersion parameter of the AgentServiceConfig custom resource (CR) now supports either "x.y" (major.minor) or "x.y.z" (major.minor.patch) formats.
```
Comment (ahardin-rh) [inline release_notes/ocp-4-11-release-notes.adoc:204]: `openshiftVersion` parameter

    `AgentServiceConfig` custom resource (CR)


per https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference

File: release_notes/ocp-4-11-release-notes.adoc (hunk 1 above)
Comment (ktothill) [reply release_notes/ocp-4-11-release-notes.adoc:202]: Updated as suggested.

File: release_notes/ocp-4-11-release-notes.adoc (hunk 2 above)
Comment (ktothill) [reply release_notes/ocp-4-11-release-notes.adoc:204]: Updated as suggested.

**Pull request merged**